src/query/block/lazy.go

```diff
@@ -21,62 +21,69 @@
 package block
 
 import (
-	"log"
 	"time"
 
 	"github.com/m3db/m3/src/query/ts"
 )
 
-// TimeTransform transforms a timestamp
-type TimeTransform func(time.Time) time.Time
+var (
+	defaultTimeTransform  = func(t time.Time) time.Time { return t }
+	defaultMetaTransform  = func(meta Metadata) Metadata { return meta }
+	defaultValueTransform = func(val float64) float64 { return val }
+)
 
-// MetaTranform transforms meta data
-type MetaTranform func(meta Metadata) Metadata
+type lazyOpts struct {
+	timeTransform  TimeTransform
+	metaTransform  MetaTransform
+	valueTransform ValueTransform
+}
 
-// ValueTransform transform a float64
-type ValueTransform func(float64) float64
+// NewLazyOpts creates LazyOpts with default values
+func NewLazyOpts() LazyOpts {
```

Comment:
nit: Can you rename it to `LazyOptions`, also maybe split it out into its own file